sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/nlp_search_space.py

```diff
@@ -39,11 +38,6 @@ def __init__(
                 for item in learning_rate_scheduler.values
             ]
 
-        if isinstance(model_name, NlpModels):
```

Comment:
it may be wise to keep this file unchanged, actually. this is just converting the model_name from enum to string if and only if it wasn't a string already. this would be good for back compat.